Manito/Manito/Screens/CreateRoom/View/CreateRoomView.swift

```diff
@@ -0,0 +1,320 @@
+//
+//  CreateRoomView.swift
+//  Manito
+//
+//  Created by 이성호 on 2023/08/02.
+//
+
+import UIKit
+
+import SnapKit
+
+protocol CreateRoomViewDelegate: AnyObject {
+    func didTapCloseButton()
+    func requestCreateRoom(roomInfo: RoomInfo, colorIndex: Int)
+}
+
+final class CreateRoomView: UIView {
+    
+    private enum CreateRoomStep: Int {
+             case inputTitle = 0, inputCapacity, inputDate, checkRoom, chooseCharacter
+    }
```

Comment:
p4;

이 친구가 좀 깊게 들어가 있네요...?? ㅎㅎ
```suggestion
    private enum CreateRoomStep: Int {
         case inputTitle = 0, inputCapacity, inputDate, checkRoom, chooseCharacter
    }
```

Reply:
[d9f2a63](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/482/commits/d9f2a63236069bd3199cd4c273eefc55f4fb9002)

맞네요 ㅎㅎ 감사합니당!